Makefile

```diff
@@ -8,14 +8,31 @@ COMPARE ?= 1
 NON_MATCHING ?= 0
 # If ORIG_COMPILER is 1, compile with QEMU_IRIX and the original compiler
 ORIG_COMPILER ?= 0
+# If COMPILER is GCC, compile with GCC instead of IDO.
+COMPILER ?= ido
```

Comment:
very minor remark but make's comparison is case-sensitive so COMPILER=GCC wouldn't work, COMPILER=gcc would
```suggestion
# If COMPILER is gcc, compile with GCC instead of IDO.
COMPILER ?= ido
```
may want to use quotes too idk